addons/sys_sounds/CfgAcreSounds.hpp

```diff
@@ -27,8 +27,8 @@ class CfgAcreSounds {
         sound = "\idi\acre\addons\sys_sounds\sounds\click.b64";
     };
 
-    // sys_sem52sl
-    class Acre_SEM52Knob {
+    // sys_sem52sl and sem70
+    class Acre_SEMKnob {
         sound = "\idi\acre\addons\sys_sounds\sounds\sem52knob.b64";
```

Comment:
There is a misspelling error in here: sound is referenced as sem52knob.b64 but the file name is `semknob.b64`